package.json

```diff
@@ -30,10 +30,13 @@
     "chokidar": "^1.4.3",
     "commander": "^2.9.0",
     "lodash.debounce": "^4.0.8",
+    "lodash.isarray": "^4.0.0",
+    "lodash.isregexp": "^4.0.1",
+    "lodash.isstring": "^4.0.1",
     "source-map-support": "^0.4.0"
   },
   "peerDependencies": {
-    "babel-core": "6.5.1 - 6.6.0 || ^6.6.x"
+    "@babel/core": "^7.0.0"
```

Comment:
@pfhayes @kmagiera 
May be it makes sense to bump major version? I mean, there is a breaking change here.

I think a lot of users has `{ babel-watch: "^2.0.2", babel-core: "^6.6.0" }` in their `package.json`. And all these users will got an error after next `npm\yarn install`.

I noticed that there is a version help in a readme, but I think it is better to prevent problem then tell how to solve one.